packages/hash/frontend/src/pages/login/login-intro.tsx

```diff
@@ -0,0 +1,145 @@
+import { useRouter } from "next/router";
```

Comment:
Possibly duplicate file for `packages/hash/frontend/src/components/auth/login/LoginIntro.tsx` (found it by searching `// const options = [`)

Reply:
Thanks for catching. It's no longer needed. I've removed it here => bd28f865103bd5a5